app/src/main/java/com/instacart/android/challenges/network/NetworkApi.java

```diff
@@ -1,17 +1,20 @@
 package com.instacart.android.challenges.network;
 
 import io.reactivex.Observable;
+import retrofit2.Call;
 import retrofit2.http.GET;
 import retrofit2.http.Path;
 
 public interface NetworkApi {
 
+  // Note : Feel free to use LiveData, RxJava, Coroutines, or whatever other tech works for you in order to interact with this API
```

Comment:
I can maybe just remove this comment, as long as we explain this to the candidate when they arrive at task 2